common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -4490,6 +4490,27 @@ unleash_old_god_effect = {
 	}
 	73 = { province_event = { id = WCOME.365 days = 360 random = 720 } }
 }
+drink_keg_brew_effect = {
+	hidden_effect = {
+		if = {
+			limit = { has_artifact_flag = keg_fullness_3 }
+			clr_artifact_flag = keg_fullness_3
+			set_artifact_flag = keg_fullness_2
+			set_name = ""
+			set_name = "Partly Empty [This.GetName]"
+		}
+		else_if = {
+			limit = { has_artifact_flag = keg_fullness_2 }
+			clr_artifact_flag = keg_fullness_2
+			set_artifact_flag = keg_fullness_1
+			set_name = ""
+			set_name = "Almost Empty [This.GetName]"
```

Comment:
you should take this into localisation file, because it doesn't work with other languages this way...

Reply:
same